### PR TITLE
fsync command

### PR DESCRIPTION
**Pull request by ilbelkyr**
Allows ops to easily synchronize the actual channel modes with the bot's view of what they should be. Could be easily extended to call this automatically when servers reset modes on netjoin; I'll look into that tomorrow.

### CODE REVIEW

File: modules/wolfgame.py [hunk 1]
```diff
@@ -229,6 +228,12 @@ def check_for_modes(cli, rnick, chan, modeaction, *target):
                     var.USERS[trgt]["moded"].remove(change)
             elif change in var.USERS[trgt]["modes"]:
                 var.USERS[trgt]["modes"].remove(change)
+    # Only sync modes if a server changed modes because
+    # 1) humans ops probably know better
```
Comment (nyuszika7h) [inline modules/wolfgame.py]: Did you mean: **human**

**Comment by jacob1** [comment]
+1, i'm for merging this as is right now :P. I really like the automated syncing during net splits since that gets annoying.

**Comment by ilbelkyr** [comment]
Now tested with actual netjoins:

    0118 052707 <@GLyKOS> IshaServ, bcode, 1, 2: Welcome to Werewolf, the popular detective/social party game (a theme of Mafia). Using the aleatoire game mode.
    0118 052707 -!- mode/#services [+m] by GLyKOS
    0118 052707 <@GLyKOS> It is now nighttime. All players check for PMs from me for instructions.
    0118 052711 <@OperServ> server_delete(): kerve.2 (72L), uplink kerve. (0 users)
    [...]
    0118 052806 -!- mode/#services [-mv bcode] by GLyKOS
    0118 052813 -!- ServerMode/#services [+m] by kerve.
    0118 052813 <@OperServ> server_add(): kerve.2 (72L), uplink kerve.
    0118 052813 -!- mode/#services [-m] by GLyKOS
    0118 052813 <@OperServ> handle_eob(): end of burst from kerve.2 (0 users)

    0118 053321 -!- mode/#services [+m] by GLyKOS
    0118 053321 <@GLyKOS> It is now nighttime. All players check for PMs from me for instructions.
    0118 053329 -!- theWolf [bcode@0::1] has quit [*.net *.split]
    0118 053329 <@OperServ> server_delete(): kerve.2 (72L), uplink kerve. (1 users)
    0118 053329 <@GLyKOS> theWolf has gone missing.
    [...]
    0118 053600 <@GLyKOS> The winner is bcode.
    0118 053600 -!- mode/#services [-mv bcode] by GLyKOS
    0118 053612 -!- ServerMode/#services [+m] by kerve.
    0118 053612 -!- theWolf (wolf) [bcode@0::1] has joined #services
    0118 053612 -!- ServerMode/#services [+v theWolf] by kerve.
    0118 053612 <@OperServ> server_add(): kerve.2 (72L), uplink kerve.
    0118 053612 -!- mode/#services [-m] by GLyKOS
    0118 053612 -!- mode/#services [-v theWolf] by GLyKOS
    0118 053612 <@OperServ> handle_eob(): end of burst from kerve.2 (1 users)

**Comment by nyuszika7h** [comment]
Looks good.

**Pull request merged**